ctapipe/io/tests/test_simteleventsource.py

```diff
@@ -331,3 +337,43 @@ def test_only_config():
 
     s = SimTelEventSource(config=config)
     assert s.input_url == Path(gamma_test_large_path).absolute()
+
+
+def test_calibscale_and_calibshift(prod5_gamma_simtel_path):
+
+    telid = 25
+
+    with SimTelEventSource(
+        input_url=prod5_gamma_simtel_path, max_events=1
+    ) as source:
+
+        for event in source:
+            pass
+
+    config = Config()
```

Comment:
We test the config system elsewhere. For this unit test, it would be much easier to just give the calib scale / calib shift as kwarg to the simtel event source.

Reply:
done